### PR TITLE
Add changelog entry for splitTask export from @wordpress/interactivity

### PR DESCRIPTION
**Pull request by westonruter**
See https://github.com/WordPress/gutenberg/pull/62665#issuecomment-2186797243

### CODE REVIEW

File: packages/interactivity/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 ## 6.1.0 (2024-06-15)
 
+### Enhancements
+
+-   Export `splitTask` function from `@wordpress/interactivity` package to facilitate yielding to the main thread. See example in [async actions](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/interactivity-api/api-reference.md#async-actions) documentation. ([#62665](https://github.com/WordPress/gutenberg/pull/62665))
+
```
Comment (cbravobernal) [inline packages/interactivity/CHANGELOG.md]:
```suggestion
```

File: packages/interactivity/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
```
Comment (cbravobernal) [inline packages/interactivity/CHANGELOG.md]:
```suggestion
### Enhancements

-   Export `splitTask` function from `@wordpress/interactivity` package to facilitate yielding to the main thread. See example in [async actions](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/interactivity-api/api-reference.md#async-actions) documentation. ([#62665](https://github.com/WordPress/gutenberg/pull/62665))
```

**Review by cbravobernal — DISMISSED** [review]
Thanks for updating the changelog!

Changes should be under ## Unreleased heading. Once packages are released it will auto-update to its correspondant version.

**Review by luisherranz — APPROVED** [review]
Perfect, thank you!

**Pull request merged**